### ISSUE TITLE
Blank Marketplace Page 

### ISSUE THREAD

**Issue by H4TERAID** [Hari-Nagarajan/fairgame#599]
**Describe the bug**
After a few minutes, chrome seems to bug out. It will stop displaying the marketplace seller frame ( Available from _These Sellers._ ) that pops up after every refresh or que, 

**To Reproduce**
Just set the _Amazon.bat delay floating point integer to 0, 1, or 2

**Expected behavior**
loads page, goes to next item, or refresh page.

**Screenshots**
[2.png](https://postimg.cc/bZcmwntq)
[1.png](https://postimg.cc/PLM3cqNT)

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Windows 10 - 19042
 - Browser: Chrome - Version 89.0.4389.90 (Official Build) (64-bit)
 - Version: 0.6.2 

**Additional context**
After the bug happens the screen is grey and needs to be clicked on to interact with anything like in the first pic.  I was mostly just curious if I get temp banned by amazon, or if chrome is bugging out due to refreshing too fast or waiting too long for Amazon to respond. The tigger seems to be amazon's  "Playing intense music" splash on the marketplace page, if this takes too long to load before the next search the bug seems to happen. No errors from Bot. Is it still working or should I stop and wait and restart?

**Comment by JudeSaint** [comment]
I actually just had the exact same issue.  Curious to see what it is.  I thought it might be a soft ban because when i tried to get to amazon live support it would tell me that something went wrong and show me their dogs lol.  Did the same thing on my phone when i tried to use the app.  After a while it just fixed itself.

**Comment by H4TERAID** [comment]
It's a soft ban. Happens so far from 1-4 seconds.

**Comment by JudeSaint** [comment]
So i need to be at 5 second intervals?

**Comment by Ampzx** [comment]
I'm even softbanned on smile.amazon.com

**Comment by JudeSaint** [comment]
I believe this is the case for me too. :(  Smile no longer shows up ... just a blank white page

**Comment by TomTom1991** [comment]
Yeah its a soft ban. But its only a cookie soft ban. Meaning opening an incognito window already does the job. As a suggestion to fix this, can you add a property for the .bat file where you can use an incognito window of chrome instead of a regular chrome session. That way you avoid that softban and you can just restart the bot when you get caught.

**Comment by Ampzx** [comment]
Except it's an account ban so when I log back on even on my phone or an
incognito window it does the same thing. Cleared cookies as well

**Comment by H4TERAID** [comment]
Alright ty guys! <3

**Comment by TomTom1991** [comment]
Not necessarily, even when it is an account ban, you can add it to your cart in incognito and then check out with your regular credentials. Tried it with a different product. So the problem solver is really the incognito window and maybe moving the login steps to the end of the script.
